lib/pbench/cli/agent/commands/results/push.py

```diff
@@ -18,19 +19,23 @@ def __init__(self, context: CliContext):
 
     def execute(self) -> int:
         tarball_len, tarball_md5 = md5sum(self.context.result_tb_name)
-        crt = CopyResultTb(
-            self.context.result_tb_name,
-            tarball_len,
-            tarball_md5,
-            self.config,
-            self.logger,
-        )
-        res = crt.copy_result_tb(
-            self.context.token, self.context.access, self.context.metadata
-        )
+        try:
+            crt = CopyResultTb(
+                self.context.result_tb_name,
+                tarball_len,
+                tarball_md5,
+                self.config,
+                self.logger,
+            )
+            res = crt.copy_result_tb(
+                self.context.token, self.context.access, self.context.metadata
+            )
+        except (RuntimeError, CopyResultTb.FileUploadError) as e:
+            click.echo(e.args[0], err=True)
+            return 1
```

Comment:
Do we really want/need this `try`/`except` block here, given that we have one already which I think does the same thing at line 97?

Reply:
OK.